packages/editor/src/store/test/selectors.js

```diff
@@ -1598,10 +1595,11 @@ describe( 'selectors', () => {
 		} );
 
 		it( 'should return true if title or excerpt have changed', () => {
-			for ( const variantField of [ 'title', 'excerpt' ] ) {
-				for ( const constantField of without(
-					[ 'title', 'excerpt' ],
-					variantField
+			const fields = [ 'title', 'excerpt' ];
+
+			for ( const variantField of fields ) {
+				for ( const constantField of fields.filter(
+					( f ) => ! f === variantField
```

Comment:
I think there is a typo on this condition, right now it is always false. I submitted a PR at https://github.com/WordPress/gutenberg/pull/42280.

Reply:
Thanks @jorgefilipecosta 🙌